tests/csv_to_geopandas_test.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+import pytest
+
+
+from eis_toolkit.conversions.csv_to_geopandas import csv_to_geopandas
+from eis_toolkit.exceptions import InvalidParameterValueException, InvalidWktFormatException, InvalidColumnIndexException
+
+parent_dir = Path(__file__).parent
+csv_path = parent_dir.joinpath("data/remote/test.csv")
+
+
+def test_csv_to_geopandas():
+    """Test csv to geopandas conversion using WKT format."""
+    indexes = [2]
+    target_EPSG = 4326
+    gdf = csv_to_geopandas(csv_path, indexes, target_EPSG)
+    assert isinstance(gdf, gpd.GeoDataFrame)
+    assert gdf.crs.to_epsg() == target_EPSG
+
+
+def test_csv_to_geopandas_using_wkt_invalid_parameter_value():
+    """Test that index out of range raises correct exception."""
+    with pytest.raises(InvalidColumnIndexException):
+        indexes = [8]
+        target_EPSG = 4326
+        gdf = csv_to_geopandas(csv_path, indexes, target_EPSG)
+        assert isinstance(gdf, gpd.GeoDataFrame)
+        assert gdf.crs.to_epsg() == target_EPSG
```

Comment:
For all the tests that are supposed to raise an Exception, you don't need the assert statements, only the call to the csv_to_geopandas